conda-store-server/conda_store_server/server/app.py

```diff
@@ -15,10 +15,13 @@
 from conda_store_server.app import CondaStore
 from conda_store_server import __version__
 
+import conda_store_server.server.dbutil as dbutil
+
 
 class CondaStoreServer(Application):
     aliases = {
         "config": "CondaStoreServer.config_file",
+        "upgrade-db": "CondaStoreServer.upgrade_db",
     }
 
```

Comment:
Add a traitlets attribute for this similar to https://github.com/jupyterhub/jupyterhub/blob/main/jupyterhub/app.py#L1297.

Reply:
Done. I realized I was using this in the config : 
```py
c.CondaStore.upgrade_db = True
```
So the `upgrade-db` traitlet property should go in `CondaStore` class, not in `CondaStoreServer`. That's fixed now.